### DISCUSSION TITLE
How to clone one voice sample and save in local, then load the local model and inference.

### DISCUSSION THREAD

**Discussion by dongyups** [coqui-ai/TTS#3995] [General Q&A]
Hi, I am currently working on a personal project cloning my voice for TTS. I followed the guidelines to synthesize examples and it worked well. 

```
### guidelines ###
from TTS.api import TTS
tts = TTS("tts_models/multilingual/multi-dataset/xtts_v2", gpu=True)

# generate speech by cloning a voice using default settings
tts.tts_to_file(text="It took me quite a long time to develop a voice, and now that I have it I'm not going to be silent.",
                file_path="output.wav",
                speaker_wav="/path/to/target/speaker.wav",
                language="en")
```

But, is it possible to clone one sample voice (speaker_wav) without fine-tuning the entire model and save it in the local directory? So that I no longer need the "speaker_wav" to generate any TTS sentences. Thanks in advance

**Comment by samarasimhapeyala** [comment]
You no need fine-tuning the entire model in any of the cases. In all cases you need to have the text to be generated and the target speaker_wav that is used for speaker characteristics to generate the audio from text. Always should have the speaker_wav file even in inference locally with saved weights.

**Comment by eginhard** [comment]
You can compute the latents and speaker embeddings once (see https://coqui-tts.readthedocs.io/en/latest/models/xtts.html#id5) and store them, then you don't need the speaker_wav.
